beta/src/styles/algolia.css

```diff
@@ -19,6 +19,9 @@
   --fds-animation-fade-in: cubic-bezier(0, 0, 1, 1);
   --fds-animation-fade-out: cubic-bezier(0, 0, 1, 1);
 }
+html {
+  scroll-behavior: smooth;
+}
```

Comment:
```suggestion
```
It works when we have it at one place alone 👀 we'd not need it inside almost all css files.